apps/web/app/layout.tsx

```diff
@@ -16,6 +17,7 @@ export default function RootLayout({
 }): JSX.Element {
   return (
     <html lang="en">
+      <NavBar />
```

Comment:
NavBar should be a inside the html `body`

Reply:
Done 🫡